crates/c-api/include/wasmtime/store.h

```diff
@@ -43,7 +43,7 @@ typedef struct wasmtime_store wasmtime_store_t;
  * \brief An interior pointer into a #wasmtime_store_t which is used as
  * "context" for many functions.
  *
- * This context pointer is used pervasively throught Wasmtime's API. This can be
+ * This context pointer is used pervasively throughout Wasmtime's API. This can be
  * acquired from #wasmtime_store_context or #wasmtime_caller_context. The
```

Comment:
I think it boils down to this:

```suggestion
 * This context pointer is used pervasively throughout Wasmtime's API. This can
 * be acquired from #wasmtime_store_context or #wasmtime_caller_context. The
```

`clang-format` wants the comments to all be a certain length.

Reply:
I went ahead and merged this suggestion to see if there are any other problems.